packages/credentials/src/V1/common.ts

```diff
@@ -8,11 +8,26 @@
 import type { ApiPromise } from '@polkadot/api'
 import { base58Decode, base58Encode } from '@polkadot/util-crypto'
 import { hexToU8a } from '@polkadot/util'
+import { ConfigService } from '@kiltprotocol/config'
 
-import type { HexString } from '@kiltprotocol/types'
-import { Caip19, Caip2, SDKErrors } from '@kiltprotocol/utils'
+import type {
+  HexString,
+  KiltAddress,
+  SharedArguments,
+  SignerInterface,
+} from '@kiltprotocol/types'
+import { Caip19, Caip2, SDKErrors, Signers } from '@kiltprotocol/utils'
 
-import type { KiltAttesterDelegationV1, KiltCredentialV1 } from './types.js'
+import { Extrinsic } from '@polkadot/types/interfaces/types.js'
```

Comment:
importing from a nested file within a package is not a thing we should be doing in modern JS. You _can_ import from a subpath declared in the package's package.json (via the `exports` field), but if your import is coming from another package and has a file extension, you're doing something wrong.

Reply:
f1a54b26e2d9d9a452962f70c01ee00cf3d6e434